scarb/tests/build_starknet_contract.rs

```diff
@@ -363,60 +363,60 @@ fn compile_starknet_contract_without_starknet_dep() {
             starknet = ">=[..]"
 
         error: Plugin diagnostic: Unsupported attribute.
-         --> [..]/lib.cairo:9:1
+         --> [..]src/lib.cairo:9:1
         #[starknet::contract]
         ^*******************^
-
+        
```

Comment:
```suggestion

```

nit: I guess the whitespaces are not needed in these empty lines :) (applies to every line in this file with extra whitespaces)